### PR TITLE
Update cli methods related to additional smart contracts

### PR DESCRIPTION
**Pull request by evgeniiz321**

### CODE REVIEW

File: src/neofs_testlib/cli/neogo/contract.py [hunk 1]
```diff
@@ -5,9 +5,9 @@
 
 
 class NeoGoContract(CliCommand):
-    def compile(
+    def contract_compile(
         self,
-        input_file: str,
+        i: str,
```
Comment (roman-khimov) [inline src/neofs_testlib/cli/neogo/contract.py:10]: Doesn't match the description below and I'd expect `i` to be used for different purpose.

**Review by roman-khimov — CHANGES_REQUESTED** [review]

File: src/neofs_testlib/cli/neogo/contract.py (hunk 1 above)
Comment (evgeniiz321) [reply src/neofs_testlib/cli/neogo/contract.py:10]: I can't replace ``` i ``` with anything else =(
here are options for this command:

```
E           OPTIONS:
E              --in value, -i value        Input file for the smart contract to be compiled
E              --out value, -o value       Output of the compiled contract
E              --verbose, -v               Print out additional information after a compiling
E              --debug value, -d value     Emit debug info in a separate file
E              --manifest value, -m value  Emit contract manifest (*.manifest.json) file into separate file using configuration input file (*.yml)
E              --config value, -c value    Configuration input file (*.yml)
E              --no-standards              do not check compliance with supported standards
E              --no-events                 do not check emitted events with the manifest
E              --no-permissions            do not check if invoked contracts are allowed in manifest
E              --bindings value            output file for smart-contract bindings configuration
```

So i have either ```in``` or ```i```, but former is the reserved python keyword, so I can use only ```i```

Comment (evgeniiz321) [reply src/neofs_testlib/cli/neogo/contract.py:10]: description updated

Comment (roman-khimov) [reply src/neofs_testlib/cli/neogo/contract.py:10]: I've thought it's just an internal name, but it's mapped directly onto CLI parameters, then yes.

Comment (evgeniiz321) [reply src/neofs_testlib/cli/neogo/contract.py:10]: Do I need to do smth else here?

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**